src/ReleaseHistory.md

```diff
@@ -6,6 +6,8 @@
 * BUGFIX: `WorkItemFiler` now logs correctly the details for `LogMetricsForProcessedModel` method [#1896](https://github.com/microsoft/sarif-sdk/issues/1896)
 * FEATURE: Add validation rule `SARIF1019`, which requires every result to have at least one of `result.ruleId` and `result.rule.id`. If both are present, they must be equal. [#1880](https://github.com/microsoft/sarif-sdk/issues/1880)
 * FEATURE: Add validation rule `SARIF1020`, which requires that the $schema property should be present, and must refer to the final version of the SARIF 2.1.0 schema.  [#1890](https://github.com/microsoft/sarif-sdk/issues/1890)
+* FEATURE: Expose Run.MergeResultsFrom(Run) to merge Results from multiple Runs using code from result matching algorithm.
+* BREAKING: Rename 'RemapIndicesVisitor' to 'RunMergingVisitor' and redesign to control how much merging occurs internally.
```

Comment:
>BREAKING [](start = 2, length = 8)

Just a consistency nit, we use markdown back-ticks to enquote things like `RemapIndicesVisitor`. A method name like `Run.MergeResultsFrom(Run)` would also typically be enquoted. This is the sort of thing that Larry and I typically scrub for before we update to NuGet (which we should do shortly), so we can clean it up for you later if you are UTG. @lgolding,I do suggest we publish 2.2.6 soon, particularly to pick up the empty `results` array. Before doing that, let's talk offline about the automatic version control provenance generation I have in mind.

Reply:
Fixed.